docs/broker.rst

```diff
@@ -127,6 +127,57 @@ The accounts must be formatted as a mapping where the ident is the key:
 }
 ```
 
+Mongo authentication store
+=========================
+
+When starting the broker you can pass a mongo connection string. Auth requests are then checked against
+the selected Database in a collection named auth_keys. Any authentication can be included within the connection string
+For example:
+
+```bash
+hpfeeds-broker -e tcp:port=20000 --exporter=0.0.0.0:9431 --auth="mongodb://127.0.0.1:27017/hpfeeds"
+```
+
+```bash
+hpfeeds-broker -e tcp:port=20000 --exporter=0.0.0.0:9431 --auth="mongodb://admin:admin@127.0.0.1:27017/hpfeeds"
+```
+
+An example Mongo Document:
+
+```json
+{
+  "identifier": "testing",
+  "secret": "secretkey",
+  "publish": [ "chan1","chan2"],
+  "subscribe": ["chan2"]
+}
+```
+
+To Find all users
+
+```bash
+mongo
+> use hpfeeds
+switched to db hpfeeds
+> show collections
+auth_key
+> db.auth_key.find()
+{ "_id" : ObjectId("5e35e5f09ba2a06adeef5be0"), "identifier" : "49be3430-4535-11ea-90b0-0242ac140004", "secret" : "q8JeUC043OYs7Mmz", "publish" : [ ], "subscribe" : [ ] }
+> 
+```
+
+To add a new user
+
+```bash
+mongo -u admin -padmid
+> use hpfeeds
+switched to db hpfeeds
+> db.auth_key.insert({"identifier": "testing", "secret": "secretkey", "publish": ["chan1", "chan2"], subscribe: ["chan2"]})
+WriteResult({ "nInserted" : 1 })
+> 
+```
+
+
 If the `aionotify` package is installed and the host os is Linux then the broker will automatically
```

Comment:
These 2 paragraphs belong with the JSON stuff so need moving above your addition.